packages/core/botpress-builtins/CHANGELOG.md

```diff
@@ -355,7 +355,6 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Features
 
-* **builtin:** storage actions ([e09af56](https://github.com/botpress/botpress/commit/e09af56))
```

Comment:
Also do you know why are some changelog entries removed?

Reply:
Because commit with this hash we removed.